crates/rpc/rpc/src/eth/api/block.rs

```diff
@@ -23,6 +23,19 @@ where
         Ok(self.client().ommers(block_id)?)
     }
 
+    pub(crate) async fn ommer_by_block_and_index(
+        &self,
+        block_id: impl Into<BlockId>,
+        index: Index,
+    ) -> EthResult<Option<RichBlock>> {
+        let block_id = block_id.into();
+        let index = usize::from(index);
+        let uncles = self.client().ommers(block_id)?.unwrap_or_default();
+        let Some(uncle_header) = uncles.get(index) else {
```

Comment:
```suggestion
        let Some(uncle_header) = uncles.skip(index).next() else {
```

Reply:
[Clippy suggests nth](https://rust-lang.github.io/rust-clippy/master/index.html#iter_skip_next), so I've used that instead :slightly_smiling_face: .